packages/fxa-content-server/app/styles/modules/_marketing.scss

```diff
@@ -7,6 +7,8 @@
     // Links have `overflow: auto` by default, which causes a scrollbar to
     // appear around the images. Get rid of that.
     overflow: inherit;
+    display: flex;
+    justify-content: space-evenly;
```

Comment:
I think you might be able remove this and do something like

```
  .marketing-link:last-child {
    margin-left: 22px;
  }
```

to give the proper spacing.

Reply:
On it